corehq/apps/analytics/tasks.py

```diff
@@ -193,6 +194,20 @@ def track_clicked_deploy_on_hubspot(webuser, cookies, meta):
     _send_form_to_hubspot(HUBSPOT_CLICKED_DEPLOY_FORM_ID, webuser, cookies, meta)
 
 
+@task(queue="background_queue", acks_late=True, ignore_result=True)
+def track_user_subscriptions_on_hubspot(webuser, properties):
+    vid = _get_user_hubspot_id(webuser)
+    if vid:
```

Comment:
all of the other hubspot property tracking follows this pattern. I assumed it was to avoid having incomplete stub contacts. you are right that it will create a new one but I believe it will only populate it with the info in that POST, so there will be an email but none of the rest of the contact information we rely on to make hubspot useful, like a name or ip address. That being said, contacts are created on sign in, so it is pretty hard to get into a state where this call will be made and the user wont already be a hubspot contact